backend/src/schema/resolvers/users.js

```diff
@@ -23,6 +23,21 @@ export const getMutedUsers = async context => {
   return mutedUsers
 }
 
+export const getBlockedUsers = async context => {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-20T16:34:48Z" title="Monday, January 20th 2020, 5:34:48 pm +01:00">Jan 20, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -48,6 +48,29 @@ export default {
         throw new UserInputError(e.message)
       }
     },
+    blockedByPostAuthor: async (_parent, params, context, _resolveInfo) => {
```


Why is it necessary that Alice can ask Bob if he blocked Mallory? Only Bob should be allowed to ask Mallory if she blocked him.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-20T16:47:25Z" title="Monday, January 20th 2020, 5:47:25 pm +01:00">Jan 20, 2020</time>_

---


We need to know if the `currentUser` has been blocked by the post author

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-20T17:09:31Z" title="Monday, January 20th 2020, 6:09:31 pm +01:00">Jan 20, 2020</time>_

---


I don't understand... The mutation checks if the `currentUser` represented by `context.user.id` has been blocked by the post author, oder?